mods/seasonal/moves.js

```diff
@@ -1609,6 +1773,28 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Psychic",
 	},
+	// urkerab
+	holyorders: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		id: "holyorders",
+		name: "Holy Orders",
+		pp: 10,
+		priority: 0,
+		flags: {},
+		onPrepareHit: function () {
+			this.attrLastMove('[still]');
+		},
+		onHit: function (target, source) {
+			this.useMove("healorder", source);
+			this.useMove("defendorder", source);
+			this.useMove("attackorder", source);
```

Comment:
You will want to pass source twice (as the target and source)

Reply:
for what reason?

Reply:
for what reason?

Reply:
To prevent any issues for not having a source + clean code.